Scripts/script-StixCreator.yml

```diff
@@ -0,0 +1,118 @@
+commonfields:
+  id: StixCreator
+  version: -1
+name: StixCreator
+script: |-
+  ''' IMPORTS '''
+  import json
+
+  from stix2 import Indicator, Bundle
+
+  indicators = []
+
+  userArgs = demisto.args().get('indicators','Unknown')
+  if isinstance(userArgs, dict) == True:
+      all_args = json.loads(json.dumps(userArgs))
```

Comment:
why you are doing dumps and then loads?

Reply:
I'm running this script from go and it passes a dictionary, when I'm using json.loads only it fails it has to convert the go dictionary to string first and only then it works.